content/docs/reference/pkg/nodejs/pulumi/pulumi/_index.md

```diff
@@ -813,7 +813,7 @@ fully qualified names, such as `pulumi:foo:a`, ..., and `pulumi:bar:a`, respecti
 </h4>
 
 
-<pre class="highlight"><code><span class='kd'>public </span>get&lt;K&gt;(key: <span class='kd'><a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String'>string</a></span>, opts?: <a href='#StringConfigOptions'>StringConfigOptions</a>&lt;K&gt;): K | <span class='kd'><a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined'>undefined</a></span></code></pre>
+<pre class="highlight"><code><span class='kd'>public </span>get&lt;K&gt;(key: <span class='kd'><a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String'>string</a></span>, opts?: StringConfigOptions&lt;K&gt;): K | <span class='kd'><a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined'>undefined</a></span></code></pre>
```

Comment:
`StringConfigOptions` and `NumberConfigOptions` aren't exported (which is really a bug in `@pulumi/pulumi`), so they don't show up in our docs. Previously, we were generating an anchor link, but those links didn't work because the anchor doesn't exist on the page.

Reply:
https://github.com/pulumi/pulumi/pull/3615